src/coreclr/ToolBox/superpmi/superpmi-shim-collector/icorjitcompiler.cpp

```diff
@@ -10,6 +10,14 @@
 
 #define fatMC                               // this is nice to have on so ildump works...
 
+CORINFO_OS g_currentOs =     CORINFO_WINNT;
```

Comment:
You should probably add this as a member to `interceptor_ICJC`, not a global.

And the default should probably be based on the `TARGET_*` defines, not always Windows. (This would be the same function as `getClrVmOs`?)